commands/web/phpstan

```diff
@@ -16,4 +16,15 @@ test -e phpstan.neon || curl -OL https://git.drupalcode.org/project/gitlab_templ
 sed -i 's/BASELINE_PLACEHOLDER/phpstan-baseline.neon/g' phpstan.neon
 # Add an empty baseline file to ensure it exists.
 test -e phpstan-baseline.neon || touch phpstan-baseline.neon
-phpstan analyse $DDEV_DOCROOT/modules/custom "$@"
+
+CONTRIB_MODULE_DIR=$(find "$DDEV_DOCROOT/modules/custom" -mindepth 1 -maxdepth 1 -type d | head -n 1)
```

Comment:
I think there a cross dependency with #112 that introduces `DRUPAL_PROJECTS_PATH` env var which should be leveraged here as well.

Reply:
That one is now in. A reroll of this would be much appreciated.

Reply:
Done